__tests__/plugin.test.js

```diff
@@ -3,6 +3,15 @@ import { transform } from '@babel/core';
 import traverse from '@babel/traverse';
 
 function runPlugin(input, opts = {}) {
+  return transform(input, {
+    filename: 'jest tests fixture',
+    compact: false,
+    plugins: [[plugin, { disablePluginVersionInjection: true }]],
+    ...opts,
+  });
+}
+
+function runPluginWithInjection(input, opts = {}) {
```

Comment:
plz don't duplicate this function, add a boolean flag instead or just use `opts`